### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by graceolivia**
Create Github Actions to run my Gradle tests and all of the `http_server_spec` suite

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@937999e9cc2425eddc7fd62d1053baf041147db7
+        with:
+          arguments: build
```
Comment (hnlee) [inline .github/workflows/gradle.yml:25]: A little confused why you need this separate workflow when you have the same steps in `ruby.yml`? Did you intend this to run your Java tests? If so, you need to do more than just `gradle build`.

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Run server
+        run: |
+          ./gradlew run &
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5.1
+      - name: Install bundler
+        working-directory: ./http_server_spec
```
Comment (hnlee) [inline .github/workflows/ruby.yml:35]: Can you just `git pull` the `http_server_spec` repo rather than copying it into this code base?

**Review by hnlee — CHANGES_REQUESTED** [review]

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (graceolivia) [reply .github/workflows/gradle.yml:25]: I added a step to run the tests!

File: .github/workflows/ruby.yml (hunk 2 above)
Comment (graceolivia) [reply .github/workflows/ruby.yml:35]: We discussed leaving this as is for now because I worked on it this morning without success

**Review by hnlee — APPROVED** [review]

**Pull request merged**